### DISCUSSION TITLE
Image upload not working in built studio

### DISCUSSION THREAD

**Discussion by aanckar** [sanity-io/sanity#3410] [Sanity Studio Support] [answered]
Hi,

I can't get image uploads to work in a built studio (works in dev). Steps to reproduce:

1. Init a blank new project (`sanity init` using `@sanity/cli@dev-preview`) and add document schema with image field (sample repo: [https://github.com/aanckar/sanity-studio-v3-test](https://github.com/aanckar/sanity-studio-v3-test))
2. Deploy studio (`sanity deploy`) (or build and serve elsewhere)
3. Open studio and attempt to upload image

For me this crashes the studio every time with an error similar to this:
```
TypeError: gfe is not a function
    at https://studio-v3-test.sanity.studio/static/main.cfa9ce3d.js:4219:34538
    at Array.find (<anonymous>)
    at KU (https://studio-v3-test.sanity.studio/static/main.cfa9ce3d.js:4219:34502)
    at https://studio-v3-test.sanity.studio/static/main.cfa9ce3d.js:4486:12756
    at https://studio-v3-test.sanity.studio/static/main.cfa9ce3d.js:4486:10430
    at Array.filter (<anonymous>)
    at _pn.getFileTone (https://studio-v3-test.sanity.studio/static/main.cfa9ce3d.js:4486:10420)
    at https://studio-v3-test.sanity.studio/static/main.cfa9ce3d.js:4486:11261
    at https://studio-v3-test.sanity.studio/static/main.cfa9ce3d.js:2781:3592
    at Object.Sve [as useMemo] (https://studio-v3-test.sanity.studio/static/main.cfa9ce3d.js:89:58029)
```

**Comment by rexxars** [comment, marked as answer]
Thanks for raising! This has been fixed in 097e5efd64cad4169ac3e51ffa44c2e0ec1a2a76 and will be part of the next release.

**Reply by rexxars** [reply]
Fixed in [v3.0.0-dev-preview.12](https://github.com/sanity-io/sanity/releases/tag/v3.0.0-dev-preview.12)
